std/container/array.d

```diff
@@ -1986,7 +1986,7 @@ if (is(Unqual!T == bool))
             _store._backend.length = _store._backend.length - 1;
         }
     }
-    /// ditto
+    ///ditto
```

Comment:
Really? It's necessary to have no whitespace here? Wouldn't that break a *lot* of existing ddocs?